champions/src/main/java/me/mykindos/betterpvp/champions/champions/builds/RandomBuild.java

```diff
@@ -12,11 +13,25 @@
 //Generate a random build
 @Singleton
 public class RandomBuild {
+
+    private final BuildManager buildManager;
+    private final ChampionsSkillManager championsSkillManager;
+
+    @Inject
+    public RandomBuild(BuildManager buildManager, ChampionsSkillManager championsSkillManager) {
```

Comment:
what i meant was, this function in particular could be a function in BuildManager or BuildRepository, I dont see any reason to create a new object here

Reply:
that makes more sense tbh, done